auth/userRouter.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const bcrypt = require("bcrypt");
+const User = require("./UserModel");
+const router = express.Router();
+
+router
+  .route('/')
+  .post((req, res) => {
+    const user = new User(req.body);
+    user.save()
+      .then(result => res.status(201).json(result))
+      .catch(err => res.status(500).json({
+        error: err.message
+      }));
+  });
+
+
+module.exports = router;
```

Comment:
You don't need this file. You've already done this within your server.js. I _think_ you might be trying to create routes for both the client and the server, is that what you're doing?